src/main/java/spoon/javadoc/internal/Javadoc.java

```diff
@@ -0,0 +1,280 @@
+/**
+ *  This file originally comes from JavaParser and is distributed under the terms of
+ * a) the GNU Lesser General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ * b) the terms of the Apache License
+ */
+package spoon.javadoc.internal;
+
+	import static spoon.javadoc.internal.JavadocInlineTag.nextWord;
+
+	import java.util.Arrays;
+	import java.util.Collections;
+	import java.util.LinkedList;
+	import java.util.List;
+	import java.util.regex.Pattern;
+	import java.util.stream.Collectors;
+
+	/**
+	* The structured content of a single Javadoc comment.
+	*
+	* <p>It is composed by a description and a list of block tags.
+	*
+	* <p>An example would be the text contained in this very Javadoc comment. At the moment of this
+	* writing this comment does not contain any block tags (such as <code>@see AnotherClass</code>)
+	*/
+	public class Javadoc {
```

Comment:
Why all the lines above have TAB after EOL? Why the last line of this file has TAB?

Reply:
I had to use an automated formatter to comply with our tough checkstyle. It did that, I don't know why. But not that it's checkstyle compliant :)